app/src/test/java/uk/ac/ebi/atlas/experimentpage/cellplot/CellPlotServiceTest.java

```diff
@@ -168,8 +171,6 @@ void throwsNullPointerExceptionIfHardCodedTestMethodIsNotSameAsDBMethod() {
                         List.of(new Gson().fromJson("{\"perplexity\": 20}", JsonObject.class))));
 
         assertThrows(NullPointerException.class, () -> {
-            //We are passing method 'umap' which is small letters, But DB has capilized method 'UMAP'
-            //This gives null List as MAP fetches from the DB, MAP keys are case-sensitive.
             subject.fetchDefaultPlotMethodWithParameterisation(experimentAccession)
                     .get(umap).getAsJsonObject();
         });
```

Comment:
This is a bad test. The method `CellPlotService::fetchDefaultPlotMethodWithParameterisation` returns an `ImmutableMap`, which will throw an NPE if you try to get a key which does not exist. This isn’t testing any business logic, but Guava’s `ImmutableMap` implementation.